firmware/Capability.cpp

```diff
@@ -0,0 +1,16 @@
+#include "Capability.h"
+
+void Capability::setup() {}
+
+void Capability::run() {}
+
+bool Capability::isEnabled() {
+    return this->enabled;
+}
+
+void Capability::enable() {
+    this->enabled = true;
+}
+void Capability::disable() {
+    this->enabled = false;
+}
```

Comment:
`this->` nije neophodan ovde, podrazumeva se da se pristupa preko `this` kada se navodi ime člana klase.